lightning/src/routing/router.rs

```diff
@@ -721,8 +730,8 @@ where L::Target: Logger {
 		// $next_hops_fee_msat represents the fees paid for using all the channels *after* this one,
 		// since that value has to be transferred over this channel.
 		// Returns whether this channel caused an update to `targets`.
-		( $chan_id: expr, $src_node_id: expr, $dest_node_id: expr, $directional_info: expr, $capacity_sats: expr, $chan_features: expr, $next_hops_fee_msat: expr,
-		   $next_hops_value_contribution: expr, $next_hops_path_htlc_minimum_msat: expr, $next_hops_path_penalty_msat: expr ) => { {
+		( $chan_id: expr, $src_node_id: expr, $dest_node_id: expr, $directional_info: expr, $capacity_sats: expr, $chan_features: expr, $max_total_cltv_expiry_delta: expr, $next_hops_fee_msat: expr,
```

Comment:
I believe you don't need `$max_total_cltv_expiry_delta` since it doesn't change.

Reply:
But it should be configurable, i.e. not always `DEFAULT_MAX_TOTAL_CLTV_EXPIRY_DELTA`, even thought this is currently only used for testing purposes?

Reply:
Ah, sorry, you right of course. Addressed in https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/cb7269de6a7ffa38aa7cdd222ab699c97e6eac8a.